tests/cypress/plugins/imageGenerator/addPlugin.js

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright (C) 2020 Intel Corporation
+ *
+ * SPDX-License-Identifier: MIT
+ */
+
+"use strict";
+
+Object.defineProperty(exports, "__esModule", {
+    value: true
+});
+exports.imageGenerator = imageGenerator;
+
+var jimp = require('jimp');
+var path = require('path');
+var fs = require('fs-extra');
```

Comment:
Let's use ``const`` and ``let`` instead of ``var`` everywhere. ``const`` is used to names that cannot be reassigned in the future, else ``let`` is used. There are some benefits of using these keywords in JS.

Reply:
Thanks. Replaced.